QUEST_LV_0100.tsv

```diff
@@ -3377,7 +3377,7 @@ QUEST_LV_0100_20150317_003376	About the possessed soul
 QUEST_LV_0100_20150317_003377	$Truth of the Suspicious Seal Stone (2)
 QUEST_LV_0100_20150317_003378	I will release the soul by defeating Stone Whale
 QUEST_LV_0100_20150317_003379	{memo X}NPCChat/FTOWER45_SQ_04 /Now I am really free
-QUEST_LV_0100_20150317_003380	Goddess Tower
+QUEST_LV_0100_20150317_003380	Goddess' Tower
```

Comment:
I guess this comes down to preference since Goddess Tower works too. 
Other alternatives:
- Tower of Goddess
- Tower of the Goddess